funfact/model/_factorization.py

```diff
@@ -79,16 +79,46 @@ def factors(self):
         '''
         return self._NodeView(
             'data',
-            list(dfs_filter(lambda n: n.name == 'tensor', self.tsrex.root))
+            list(dfs_filter(lambda n: n.name == 'tensor' and
+                            n.abstract.optimizable, self.tsrex.root))
         )
 
     @factors.setter
     def factors(self, tensors):
         for i, n in enumerate(
-            dfs_filter(lambda n: n.name == 'tensor', self.tsrex.root)
+            dfs_filter(lambda n: n.name == 'tensor' and
+                       n.abstract.optimizable, self.tsrex.root)
         ):
             n.data = tensors[i]
 
+    @property
+    def all_factors(self):
+        '''A flattened list of all factors in the model.
+
+        Examples:
```

Comment:
Could you please update the example in the docstring to reflect the difference between `factors` and `all_factors`?